pkg/workloadmeta/collectors/internal/containerd/image_sbom_trivy.go

```diff
@@ -42,6 +42,8 @@ func (c *collector) startSBOMCollection(ctx context.Context) error {
 	trivyConfiguration.ContainerdAccessor = func() (cutil.ContainerdItf, error) {
 		return c.containerdClient, nil
 	}
+	trivyConfiguration.CheckDiskUsage = config.Datadog.GetBool("container_image_collection.sbom.check_disk_usage")
+	trivyConfiguration.MinAvailableDisk = uint64(config.Datadog.GetSizeInBytes("container_image_collection.sbom.min_available_disk"))
```

Comment:
It seems that the disk usage check feature that this PR implements has been made as collector (containerd, docker…) agnostic as possible.
Still, there is this part which is in a `containerd` specific file.
So, #16068 will need to be amended to add a similar piece of code in the `docker` specific part around [here](https://github.com/DataDog/datadog-agent/pull/16068/files#diff-b74c425242947644f6c5c8d1661eb8189968062f328aa08331be7cdf86bf60bbR46-R50).

Instead, would it make sense to move those two lines inside `trivy.DefaultCollectorConfig` (somewhere around [here](https://github.com/DataDog/datadog-agent/pull/16379/files#diff-cf6f6bf05119fdde25a68cca6cc05b0a80a9ab57af9eff28ccf1f08d602ad124R83)) ?

Reply:
It would make sense, and honestly it is what I would prefer to do. The reason I did not do it was because we are already doing config parsing in the workloadmeta portion before we create the trivy instance, where we are checking cache directory and settings. So, I figured this would keep us using the same pattern.

But, this comment is enough justification for me to move it!